plugins/module_utils/network/eos/rm_templates/ospfv3.py

```diff
@@ -257,9 +261,29 @@ def _tmplt_ospf_timers_throttle(config_data):
         return command
 
 
+def _tmplt_ospf_bfd(config_data):
+    if os_version < "4.23":
+        command = "bfd all-interfaces"
+    else:
+        command = "bfd default"
+    return command
+
+
+os_version = ""
+
+
 class Ospfv3Template(NetworkTemplate):
-    def __init__(self, lines=None):
-        super(Ospfv3Template, self).__init__(lines=lines, tmplt=self)
+    def __init__(self, lines=None, module=None):
+        global os_version
```

Comment:
Not a big fan of global variables, but I don't have a better solution than this since the methods have always been kept outside of the class.